### ISSUE TITLE
I get the following error when using the url without CMS:

### ISSUE THREAD

**Issue by MohammadSamandar** [kapt-labs/django-check-seo#58]
'cms_tags' is not a registered tag library. Must be one of:
admin_list
admin_modify
admin_urls
cache
i18n
jalali_tags
l10n
log
poll_extras
render_partial
sorl_thumbnail
static
thumbnail
tz

**Comment by MohammadSamandar** [comment]
I deleted this line {% load cms_tags i18n static %} from the default.html but it didn't work

**Comment by corentinbettiol** [comment]
Try with `{% load i18n static %}`, it should work.

**Comment by MohammadSamandar** [comment]
it's working now. thank you bro.

**Comment by corentinbettiol** [comment]
I will reopen the issue, since I need to fix this problem; the cms_tags templatetags may not be necessary at all and may be removed soon :)

**Comment by MohammadSamandar** [comment]
I now realized that this problem is solved only locally and the problem still remains in production mode. In production mode, the server installs the packages from the requirements.txt file, and the changes I make have no effect.

**Comment by MohammadSamandar** [comment]
I manually added this package as a program to the project, but only the "No problem was found on the page!
on the public page" message is displayed

**Comment by corentinbettiol** [comment]
> I now realized that this problem is solved only locally and the problem still remains in production mode. In production mode, the server installs the packages from the requirements.txt file, and the changes I make have no effect.

@MohammadSamandar Hi! I released a new version of django-check-seo (0.6.3) with the fix of this issue!

You should be able to install & use django check seo on your prod without any issue now :)

----

> I manually added this package as a app to the project, but only the "No problem was found on the page!
on the public page" message is displayed

If you're not using django-cms you will need to create the "check seo" links yourself on your pages templates. Here's a snippet to get you started (I have not tested it, it *may* work out of the box):

```jinja2
{% if request.user.is_superuser %}
  <a href="/django-check-seo/?page={{ request.path }}" target="_blank">Check SEO</a>
{% endif %}
```
This code will add a "check seo" link in your page if you're a superuser, that will display the Django Check SEO page for the current url.
